src/hls-adapter.js

```diff
@@ -149,6 +149,9 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
         adapterConfig.hlsConfig.startPosition = config.playback.startTime;
       }
     }
+    if (Utils.Object.hasPropertyPath(config, 'playback.useNativeTextTrack')) {
+      adapterConfig.subtitleDisplay = Utils.Object.getPropertyPath(config, 'playback.useNativeTextTrack');
```

Comment:
just add it on `adapterConfig.hlsConfig`.

Reply:
It's not an hls.js config option - it's an API. I wanted to add it as config first but it doesn't get set on hls.js config...